frontend/Components/Login.js

```diff
@@ -0,0 +1,210 @@
+import {
+    Text,
+    View,
+    StyleSheet,
+    Image,
+    TextInput,
+    Pressable,
+    Platform
+} from 'react-native';
+import React from 'react';
+
+
+export const Login = ({ navigation}) => {
+    return (
+        <View style={Styles.device}>
+            <View style={Styles.container}>
+                <Image 
+                    source={logoPath}
+                    style={Styles.logo}
+                /> 
+                <View style={Styles.titleContainer}>
+                    <Text 
+                        style={Styles.title}
+                        adjustsFontSizeToFit
+                    >
+                        Log in
+                    </Text>
+                </View>
+                <MakeTextInput type={"Email"}/>
+                <MakeTextInput type={"Password"}/>
+                <Pressable
+                    style={Styles.button}
+                    onPress={() => checkCredentials(navigation)}
+                >
+                    <Text style={Styles.buttonText}>Log in</Text>
+                </Pressable>
+                <Pressable 
+                    style={Styles.farLink}
+                    onPress={() => forgotPassword(navigation)}>
+                    <Text style={Styles.smallLink}>Forgot your password?</Text>
+                </Pressable>
+                <View style={Styles.lastLine}>
+                    <Text style={{fontSize: 16}}>
+                        Don't have an account? 
+                        <View style={Styles.inLineLink}>
+                        <Pressable
+                            onPress={() => goToSignUp(navigation)}
+                            >
+                            <Text style={Styles.mediumLink}>
+                                Sign Up
+                            </Text>
+                        </Pressable>
+                        </View>
+                    </Text>
+                </View>
+            </View>
+            {Platform.OS == 'web' &&
+            <View  style={Styles.imgContainer}>
+            <Image
+                source={require('../assets/images/themeImage.png')}
+                style={Styles.themeImage}
+            />
+            </View>}
```

Comment:
I think this is not the best way to handle this, since we don't really care what the platform is but what we care about is the screen size. 
Try to minimize your browser window width to a smaller size, this should be equivalent to using a phone screen but instead you will find the image still there and it's not the best looking.

I'm not sure how this could be handled in react-native but in web it's handled by using media-queries, try to lookup it up, if you couldn't find out how to do it, just ignore it, it's not a big deal.